packages/unit-testing/stubs/crypto/SecretKey.js

```diff
@@ -0,0 +1,70 @@
+define([], function () {
+    /**
+     * Returns a new instance of SecretKey used for hmac, cipher and decipher;
+     * Use createSecretKey to create an instance;
+     *
+     * @class
+     * @classdesc Encapsulates the handle to the key. The handler does not store the key value. It points to the key stored within the NetSuite system. The GUID or secret is also required to find the key.
+     * @param {String} guid
+     * @param {String} secret
+     * @param {String} encoding
+     * @constructor
+     * @protected
+     *
+     * @since 2015.2
+     */
+    function SecretKey() {
+
+        /**
+         * The GUID associated with the secret key.
+         * @name SecretKey#guid
+         * @type {string}
+         * @readonly
+         * @throws {SuiteScriptError} READ_ONLY_PROPERTY when setting the property is attempted
+         *
+         * @since 2015.2
+         */
+        this.guid = undefined;
+        /**
+         * The secret for the key
+         * @name SecretKey#secret
+         * @type {string}
+         * @readonly
+         * @throws {SuiteScriptError} READ_ONLY_PROPERTY when setting the property is attempted
+         *
+         * @since 2015.2
+         */
+        this.secret = undefined;
```

Comment:
helpdocs: https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=section_4358549582.html

Tree view shows SecretKey.secret but table view shows SecretKey.password

Adding comment to track inconsistency in help docs.

Reply:
![image](https://user-images.githubusercontent.com/20229175/203376142-e8414cfe-2266-4296-92c3-38c9b9d30477.png)